lib/matplotlib/axes/_axes.py

```diff
@@ -2218,7 +2218,7 @@ def bar(self, *args, **kwargs):
             if yerr is not None:
                 yerr = self.convert_yunits(yerr)
 
-        x, height, width, y, linewidth = np.broadcast_arrays(
+        x, height, width, y, ll = np.broadcast_arrays(
```

Comment:
What does the linewidth in the bar plot have to do with the ffmpeg animation? And if this is changed, why is `linewidth` used in the remainder of the function, and not `ll`?

Reply:
Oh shoot.  Somehow injected a test in there.  Thanks for catching that.  I'll fix ASAP.